src/SamplesApp/SamplesApp.UITests/Windows_UI_Core/CoreDispatcher/CoreDispatcher_Tests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Core.CoreDispatcher
+{
+  [TestFixture]
```

Comment:
This test should be a runtime test (UI Tests are expensive to run). Could you transfer it there? 

https://github.com/unoplatform/uno/tree/master/src/Uno.UI.RuntimeTests/Tests